webpack/webpack.prod.config.js

```diff
@@ -162,6 +160,8 @@ module.exports = Merge.smart(commonConfig, {
       REDDIT_URL: null,
       APPLE_APP_STORE_URL: null,
       GOOGLE_PLAY_URL: null,
+      NEW_RELIC_APP_ID: null,
```

Comment:
Put these at the bottom as they're not in the dev version/I wasn't sure if they should be.